tests/kafkatest/services/connect.py

```diff
@@ -140,6 +141,12 @@ def pause_connector(self, name, node=None):
     def resume_connector(self, name, node=None):
         return self._rest('/connectors/' + name + '/resume', method="PUT")
 
+    def list_connector_plugins(self, node=None, retries=0, retry_backoff=.01):
+        return self._rest_with_retry('/connector-plugins/', node=node, retries=retries, retry_backoff=retry_backoff)
+
+    def validate_config(self, connector_type, validate_request, node=None, retries=0, retry_backoff=.01):
+        return self._rest_with_retry('/connector-plugins/' + connector_type + '/config/validate', validate_request, node=node, method="PUT", retries=retries, retry_backoff=retry_backoff)
```

Comment:
Same here re: `_rest_with_retry` since validation just uses a temporary connector instance to test the config.